src/codecs/mp3utils.c

```diff
@@ -397,8 +397,18 @@ static size_t id3v22_parse_frame(Mix_MusicMetaTags *out_tags, struct mp3file_t *
 
     if (size < ID3v2_BUFFER_SIZE) {
         read_size = MP3_RWread(src, buffer, 1, size);
+        if (read_size < size) {
+            SDL_Log("id3v22_parse_frame (2): Buffer size that left is too small (%u < %u)", (unsigned int)read_size, (unsigned int)size);
```

Comment:
I don't understand these error changes, which is fine, but I do have a wording suggestion.

`Buffer size that left is too small` should be `Buffer size that is left is too small`. Or perhaps it would be better said as `Not enough buffer size remaining for read` or something.

Reply:
This error is about the reading from the external file, and when we wanted to read, for example, 25 bytes, but had read 13 in fact. That means an error of an unexpected file end.

Reply:
I think I'll rephrase them into `unexpected end of the file` which is a true explanation of the error.

Reply:
Done.